.gitignore

```
@@ -19,6 +18,10 @@
 !.eslintrc.yaml
 !.cpplint
 
+# Prohibit node_modules in the root dir but allow them everywhere else.
+/node_modules
+!**/node_modules
```

🔥

---

I guess that is meant as something positive?

---

If 🔥 means "hot" or "you are on fire" I agree, this looks great IMO.

---

```suggestion
!**/node_modules/**
```
and move below https://github.com/nodejs/node/blob/53580b6cdc35668072b0fe8a9faaf9e015fdeaf1/.gitignore#L42

I tested this with `git ls-files -i --exclude-standard` which shows files that are both tracked and covered by an ignore rule, and this change adds 61 files to that list (should be just 3 .log files stuck in V8)

---

P.S. took this part into https://github.com/nodejs/node/pull/27954
so addressing my comment is nice-to-have but non-blocking